app/src/main/kotlin/github/com/raccok/dota2androidapp/sources/remote/Dota2ApiService.kt

```diff
@@ -2,13 +2,13 @@ package github.com.raccok.dota2androidapp.sources.remote
 
 import github.com.raccok.dota2androidapp.api.HeroDataModelResponse
 import io.reactivex.Observable
-import retrofit2.http.GET
-import retrofit2.http.Query
 import retrofit2.Retrofit
 import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory
 import retrofit2.converter.gson.GsonConverterFactory
+import retrofit2.http.GET
+import retrofit2.http.Query
 
-interface DotaHerosApiService {
+interface Dota2ApiService {
```

Comment:
Well I wouldn't make this an `interface` in the first place. I consider this a static utility class that can be used to query any data available in the official Dota 2 API that interests us. I don't think we will go much beyond querying heroes/items/matches/player data so in my opinion this can all be packed together in one static class without filling it too much

Reply:
If that is the case, can we rename it "Dota2OfficalAPIService"?